src/pages/settings/panes/Plugins.tsx

```diff
@@ -0,0 +1,121 @@
+import { Check } from "@styled-icons/boxicons-regular";
+import { observer } from "mobx-react-lite";
+import styled from "styled-components";
+
+import styles from "./Panes.module.scss";
+import { Text } from "preact-i18n";
+
+import { useApplicationState } from "../../../mobx/State";
+
+import Button from "../../../components/ui/Button";
+import { CheckboxBase, Checkmark } from "../../../components/ui/Checkbox";
+import Tip from "../../../components/ui/Tip";
+
+// Just keeping this here for general purpose. Should probably be exported
+// elsewhere, though.
+interface Plugin {
+    namespace: string;
+    id: string;
+    version: string;
+    enabled: boolean | undefined;
+}
+
+const CustomCheckboxBase = styled(CheckboxBase)`
+    margin-top: 0 !important;
+`;
+export interface CheckboxProps {
+    checked: boolean;
+    disabled?: boolean;
+    onChange: (state: boolean) => void;
+}
+function PluginCheckbox(props: CheckboxProps) {
+    // HACK HACK HACK(lexisother): THIS ENTIRE THING IS A HACK!!!!
+    /*
+        Until some reviewer points me in the right direction, I've resorted to
+         fabricating my own checkbox component.
+       "WHY?!", you might ask. Well, the normal `Checkbox` component can take
+         textual contents, and *also* adds a `margin-top` of 20 pixels.
+        We... don't need that. At all. *Especially* the margin. It makes our card
+         look disproportionate.
+        
+        Apologies, @insert!
+    */
+    return (
+        <CustomCheckboxBase disabled={props.disabled}>
+            <input
+                type="checkbox"
+                checked={props.checked}
+                onChange={() =>
+                    !props.disabled && props.onChange(!props.checked)
+                }
+            />
+            <Checkmark checked={props.checked} className="check">
+                <Check size={20} />
+            </Checkmark>
+        </CustomCheckboxBase>
+    );
+}
```

Comment:
I'll let this slide for now since components are in the middle of being migrated so this will be fixed as a by-product.